app/pb_kits/playbook/pb_user/_user.html.erb

```diff
@@ -1,11 +1,22 @@
 <%= content_tag(:div,
     id: object.id,
     data: object.data,
-    class: object.classname(object.kit_class)) do %>
-  <%= object.avatar %>
+    class: object.classname) do %>
+  <% if object.avatar_url.present? || object.avatar %>
+    <%= pb_rails("avatar", props: {
+      name: object.name,
+      size: object.avatar_size,
+      image: {
```

Comment:
Instead of this hash, should it be `image_url: object.avatar_url` ?

Reply:
We're being explicit with this one to reach into the hash and get the image url for it to be then used as image_url on the front-end.